File: Question3.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string name;
    cout << "Enter student's name: ";
    getline(cin, name);

    double lab_grade;
    cout << "Enter lab grade: ";
    cin >> lab_grade;

    double midterm_grade;
    cout << "Enter midterm grade: ";
    cin >> midterm_grade;

    double final_grade;
    cout << "Enter final grade: ";
    cin >> final_grade;

    double last_score = (lab_grade * 0.25) + (midterm_grade * 0.35) + (final_grade * 0.40);

    cout << "Student: " << name << endl;
    cout << "Lab Grade: " << lab_grade << endl;
    cout << "Midterm Grade: " << midterm_grade << endl;
    cout << "Final Grade: " << final_grade << endl;
    cout << "Last Score: " << last_score << endl;

    return 0;
}

File: Question3.py
#Question3:
name = input("Enter the student's name: ")
lab = float(input("Enter the lab grade: "))
midterm = float(input("Enter the midterm grade: "))
final = float(input("Enter the final grade: "))
last_score = lab * 0.25 + midterm * 0.35 + final * 0.4
print("Name:", name)
print("Lab:", lab)
print("Midterm:", midterm)
print("Final:", final)
print("Last score:", last_score)

